### PR TITLE
fix #119917 dafflix.net

### LINKED ISSUE #119917: www.dafflix.net
### Issue URL (Ads)
 [https://www.dafflix.net/dizi/rick-and-morty-izle/sezon-4/bolum-2](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.dafflix.net%2Fdizi%2Frick-and-morty-izle%2Fsezon-4%2Fbolum-2)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.0.163
Browser: | Chrome
Stealth mode options: | Send Do-Not-Track header, <br>Strip URLs from tracking parameters
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish

### PR DESCRIPTION
**Pull request by ghost**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/en/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

#### To avoid invalid pull requests, please check and confirm following terms

- [ ] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

#### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### What issue is being fixed?

#119917

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

0. **DO NOT** upload screenshots with sexually explicit material on GitHub directly.\
 Instead, upload it to a third-party image hosting and post the link to it without preview (!) here.\
 Also, mention if the link leads to NSFW content;

1. Add screenshots of the problem. You can drag and drop images or paste them from clipboard

    Use `<details> </details>` tag to hide screenshots under the spoiler;

2. Describe the issue in detail until it is absolutely clear from the screenshot what the problem is

    You can also indicate any other information that you think the developers should know.

**Warning:** Please remove personal information before uploading screenshots!

1. Your comment

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

</details>

<details>

<summary>Screenshot N:</summary>

</details>

### Terms

- [ ] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -66,6 +66,15 @@ fullfilmcidayim.com,dizispeed.com,babafullfilmizleme.com###adStarter
...
 !
+||dafflix.com/olley.mp4
+||dafflix.net/yenipop2.js
+dafflix.net##.player-ustu-metin > a[rel="nofollow"]
+dafflix.net##body > div[data-type="pgskin"]
+dafflix.net##.right.floated > div[data-gets] > a[href][target="_blank"] > img
+||dafflix.net/akk.gif
+||dafflix.net/2rek.webp
+dafflix.net##.rek-wrap > *:not(b)
+dafflix.net##.rek-0
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:77]: I don't see
```
dafflix.net##.player-ustu-metin > a[rel="nofollow"]
dafflix.net##body > div[data-type="pgskin"]
||dafflix.net/akk.gif
||dafflix.net/2rek.webp
```
For the rest of the rules, change the domain to `dafflix.*`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (dcxstq2) [reply TurkishFilter/sections/specific.txt]: > I don't see

I know, maybe they'll use it again later. Just restore.

Comment (zloyden) [reply TurkishFilter/sections/specific.txt]: @dcxstq2 what about the rules above?

Comment (dcxstq2) [reply TurkishFilter/sections/specific.txt]: Rules should only be deleted if the website is permanently down.

Comment (zloyden) [reply TurkishFilter/sections/specific.txt]: I mean that the rules above are not used and we can remove them from the pull request.
We only add necessary rules.

Comment (dcxstq2) [reply TurkishFilter/sections/specific.txt]: removed

**Review by zloyden — APPROVED** [review]

**Pull request merged**